src/pages/iou/IOUModal.js

```diff
@@ -452,6 +452,7 @@ class IOUModal extends Component {
                                                 hasMultipleParticipants={this.props.hasMultipleParticipants}
                                                 onAddParticipants={this.addParticipants}
                                                 onStepComplete={this.navigateToNextStep}
+                                                safeAreaPaddingBottomStyle={safeAreaPaddingBottomStyle}
                                             />
```

Comment:
Why should we pass?
Adding `safeAreaPaddingBottomStyle` to style of `AnimatedStep` should work right?

Reply:
Also `IOUParticipantsPage`  renders `IOUParticipantsSplit`  or `IOUParticipantsRequest` which has their own `SafeAreaConsumer`,  wont it becomes unnecessary?

Reply:
> Adding `safeAreaPaddingBottomStyle` to style of `AnimatedStep` should work right?

If we do that, the main issue will happen:
`bottom home bar on iOS doesn’t have a transparent background`
Because bottom padding applied to parent view which has scroll view as a child.